idaes/core/base/property_base.py

```diff
@@ -504,6 +505,68 @@ def report(self, index=(0), true_state=False, dof=False, ostream=None, prefix=""
 
         ostream.write("\n" + "=" * max_str_length + "\n")
 
+    def get_port_reference_name(self, component_name, port_name):
+        """
+        Get the standard name of a "port reference", the component
+        accessed on the port by "port_name.component_name".
+
+        Args:
+            component_name - name of port member of interest (str)
+            port_name - name of Port object (str)
+
+        Returns:
+            str with name for Reference used for Port member
+        """
+        return f"_{component_name}_{port_name}_ref"
+
+    def build_port(
+        self,
+        port_name,
```

Comment:
With the new API, `port_name` seems not to be used. Can this argument be removed?